.github/workflows/nightly_build.yml

```diff
@@ -0,0 +1,36 @@
+name: Nightly Build
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Run this workflow every day at midnight
+    - cron: '0 0 * * *'
+
+jobs:
+    setup-and-test:
+        uses: ./.github/workflows/setup-python-environment.yml
+        with:
+          os: "ubuntu-latest"
+          python-version: "3.8"
+          git-ref: "develop"
+        secrets: inherit
+
+    publish-python-package:
+        needs: setup-and-test
+        uses: ./.github/workflows/publish_to_pypi_nightly.yml
+        secrets: inherit
+
+    wait-for-package-release:
+        runs-on: ubuntu-latest
+        needs: publish-python-package
+        steps:
+          - name: Sleep for 4 minutes
+            run: sleep 240
+            shell: bash
+
+    publish-docker-image:
+        needs: wait-for-package-release
+        uses: ./.github/workflows/publish_docker_image_nightly.yml
```

Comment:
This should not have the nightly suffix I think?

Reply:
I think we also need to pass the normal release config in the existing release workflow?

Reply:
No need for the normal release config to be passed in for the existing release workflow as it's specified explicitly as the default in `publish_docker_image.yml`. But the nightly suffix can be deleted, yes.